20180913/AGENDA.md

```diff
@@ -0,0 +1,21 @@
+# Sardana Follow-up Meeting - 2018/09/13
+
+To be held on 2018/09/13 at 14:00
+
+# Agenda
+
+1. Presentation
+    * Any volunteer?
```

Comment:
Maybe, together with @daneos - what do you think?, we could make a short presentation about the release process steps. When I say presentation, just go through the [release process steps](https://github.com/sardana-org/sardana/blob/develop/doc/how_to_release.md). present our impressions about the last release, etc.

IMHO the step from AOB deserves to be moved to the 1st point:)

Reply:
Sure, we can move this to be the first point.